domains/mobilecoin/validation.go

```diff
@@ -65,6 +70,10 @@ func GenerateAssetId(assetKey string) crypto.Hash {
 	switch assetKey {
 	case MobileCoinChainBase:
 		return MobileCoinChainId
+	case "0":
+		return MobileCoinChainId
```

Comment:
There should only be one asset key with the same asset id, don't use two different keys for one asset.

Reply:
Fixed